plenum/common/func_utils.py

```diff
@@ -0,0 +1,28 @@
+
+
+def flatten(some_list):
+    """
+    [[a,b],[c,d]] => [a,b,c,d]
+    """
+    for inner_list in some_list:
+        for item in inner_list:
+            yield item
+
+
+def decons(some_list):
```

Comment:
I used it earlier, but then changed implementation. But I decided to leave it because I need it sometimes.